overrides/engine/ui/ingame/deathScreen.ui

```diff
@@ -106,27 +117,67 @@
                       "cc": "cell 0 1"
                     },
                     "contents": [
+                      {
+                        "type": "UIImage",
+                        "image": "LightAndShadowResources:spadesIcon",
+                        "cc": ""
+                      },
                       {
                         "type": "UILabel",
                         "text": "Spades Team",
+                        "family": "spadesTeamLabel",
+                        "cc": ""
+                      },
+                      {
+                        "type": "UISpace",
+                        "cc": ""
+                      },
+                      {
+                        "type": "UISpace",
+                        "cc": ""
+                      },
+                      {
+                        "type": "UISpace",
                         "cc": ""
                       },
                       {
                         "type": "UISpace",
                         "cc": ""
                       },
                       {
-                        "type": "ScrollableArea",
-                        "id": "gameDetailsScrollableArea",
+                        "type": "UISpace",
+                        "cc": ""
+                      },
+                      {
+                        "type": "UISpace",
+                        "cc": ""
+                      },
+                      {
+                        "type": "UISpace",
+                        "cc": ""
+                      },
+                      {
+                        "type": "UISpace",
+                        "cc": ""
+                      },
```

Comment:
The only thing I didn't manage to reduce is this code. I tried many things, but non prove effective.

What I am doing here, is filling up the row with empty spaces, so the width of the row is forced to be divided into many little spaces. 

My initial tries revolved around the setting of 1 UISpace, and tried to set the desired widths via code, my tries - which all ended up at having 3 equally divided columns - were:
- Set a "width" at the UISpace
- Set ColumnConstrains at the MigLayout
- Place them in a RowLayout, and tried to modify that
- Use "span 8" at the UISpace to span it 8 cells
- Use "cell [column] [row] [width] [height]" where the "[ ]" contain numbers, the cells were arranged correctly, but the width and height didn't change
- Removed the "dock south" from the second MigLayout (L167) and placed it at the bottom with a width 3 and re-done the above methods, all failed

These are the methods that I can recall trying, and this above one was _sadly_ the only one that worked.

Reply:
Huh, interesting. Maybe be worth filing a bug report or feature enhancements on this into the TeraNUI repo?

Reply:
I could do that, yes. I just had a thought that this might occur only with UISpaces, considering the above methods mentioned were things that worked under others circumstances (uncertain what the others consist of).